server/utils/terminal/terminal_run.py

```
@@ -203,3 +213,34 @@ def terminal_run(
         stderr=stderr
     )
 
+
+def _check_guardrails(args: Union[str, List[str]]) -> Optional[CompletedProcess]:
```

Ruff (0.15.11)

[warning] Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)